src/main/java/com/igot/cb/community/kafka/consumer/Consumer.java

```diff
@@ -56,6 +62,43 @@ public void upateUserCount(ConsumerRecord<String, String> data) {
     }
   }
 
+  @KafkaListener(groupId = "${kafka.topic.community.group}", topics = "${kafka.topic.community.post.count}")
```

Comment:
these values should be added in application.properties ??
do make sure the group and topic used in other service are used here too...